README.md

```diff
@@ -50,22 +62,22 @@ pipeline:
     observationTimeStamp: '2018-10-11T13:01:02-08:00'
 ```
 
-Step 1 involves copying the source files into a folder:
+First we copy all the source files into a folder:
 ```bash
-mkdir ~/inputs
-cp orthomosaic.tif ~/inputs
-cp plot_shapes.* ~/inputs
-cp experiment.yaml ~/inputs
+mkdir /inputs
```

Comment:
Docker run requires an absolute path. We can still give directions which work with the current working directory if we specify a Docker run command like this:

```bash
docker run --rm -v "${PWD}/inputs:/sources <etc.>
```

It's just the tilde that breaks things.

Reply:
Creating directories in root directory (`/`) gave me a permissions error if I don't use `sudo`.

- [ ] Create issue to use `"$(pwd)/inputs"` instead of `/inputs` (non-blocking)

Reply:
@julianpistorius No need for a PR. According to @dlebauer comment it needs to be done now: https://github.com/AgPipeline/drone-makeflow/pull/4/files#r468945497

Reply:
Updated to use ${PWD}